aiperf/post_processors/telemetry_results_processor.py

```
@@ -0,0 +1,96 @@
+                        header = (
+                            f"{metric_display} (GPU {gpu_index}, {gpu_uuid[:8]}...)"
+                        )
+
+                        unit = self._metric_units.get(metric_name, "")
+
+                        result = telemetry_data.get_metric_result(
+                            metric_name, tag, header, unit
+                        )
+                        results.append(result)
+                    except Exception:
+                        continue
```

Ruff (0.13.1)

`try`-`except`-`continue` detected, consider logging the exception

(S112)